bridge/bridge.go

```diff
@@ -427,6 +436,18 @@ func (b *Bridge) shouldRemove(containerId string) bool {
 	return false
 }
 
+func (b *Bridge) markContainerAsDying(containerId string) {
+	// cleanup after CleanupDyingTtl
+	for containerId, t := range b.dyingContainers {
```

Comment:
I suggest we extract this cleanup into separate function and invoke it from `add()` first thing. This is to make sure that before we add a container - we have checked and cleaned-up dying ones. Otherwise remove may not be called that often and clean-up will not take place in time

Reply:
In my opinion, this is not what we want, because once we got a `kill` or `die` event we need to make sure that that container will not be re-added on the `sync`.  Cleanup was added only to not keep populating `dyingContainers` map. Also, we must set `CleanupDyingTtl` much bigger than `resyncInterval`

Reply:
`Sync` already clean up block https://github.com/auto1-oss/registrator/pull/13/files#diff-b63fd83ac48781980fae69dec381d9a060c40c2d94364390b01d02c8ed4b09abR130 . It would make sence to purge expired  `dyingContainers` there as well.